src/Compute/Compute/help/Set-AzVmssDiskEncryptionExtension.md

```diff
@@ -274,7 +274,12 @@ Accept wildcard characters: False
 ```
 
 ### -VolumeType
-Type of the volume (OS or Data) on which to perform encryption operation. This parameter is required when encrypting Linux virtual machines, and must be set to a value supported by the Linux distribution. This parameter can be omitted when encrypting Windows virtual machines.
+Specifies the type of virtual machine volumes on which to perform encryption operation: OS, Data, or All. If this parameter is omitted, the operation defaults to All.
```

Comment:
No default for Linux, just Windows.

Reply:
Removed the final sentence, moved default into to Windows only.